packages/client/src/common/components/Login/Login.tsx

```diff
@@ -36,58 +38,84 @@ const Login = () => {
 		if (response.data.status === 200) {
 			window.localStorage.setItem('access_token', response.data.token);
 			setIsLoggedIn(true);
+			setIsLoading(false);
 			//navigate to home
```

Comment:
ModalContainer should be here after successfully logged in, make sure that you've the last updated from main.